authority/voting/client/client.go

```diff
@@ -154,23 +157,29 @@ func (p *connector) initSession(ctx context.Context, doneCh <-chan interface{},
 		}
 	}()
 
-	var ad []byte
-	if signingKey != nil {
-		ad = signingKey.Bytes()
+	peerLinkPublicKey, err := wire.DefaultScheme.PublicKeyFromPemFile(filepath.Join(p.cfg.DataDir, peer.LinkPublicKeyPem))
```

Comment:
these changes break katzen, 
which serializes its configuration into the encrypted statefile
It would be better to have the right types in the Config that can be serialized to CBOR
and that way the client can also be flexible in where it reads the PKI root keys from (i.e. they may be embedded in a binary or stored in another format)